skfda/misc/metrics/_lp_distances.py

```diff
@@ -3,21 +3,22 @@
 from __future__ import annotations
 
 import math
-from typing import Optional, TypeVar, Union
+from typing import TYPE_CHECKING, Final, TypeVar
 
 import numpy as np
-from typing_extensions import Final
 
 from ...representation import FData
-from ...typing._metric import Norm
 from ...typing._numpy import NDArrayFloat
 from ._lp_norms import LpNorm
 from ._utils import NormInducedMetric, pairwise_metric_optimization
 
+if TYPE_CHECKING:
+    from ...typing._metric import Norm
+
 T = TypeVar("T", NDArrayFloat, FData)
 
 
-class LpDistance(NormInducedMetric[Union[NDArrayFloat, FData]]):
+class LpDistance(NormInducedMetric[NDArrayFloat| FData]):
```

Comment:
I think a space is missing here before the pipe operator. I am actually surprised this is not flagged by Ruff or WPS.